manifests/server.pp

```diff
@@ -581,12 +581,24 @@
 
   if $extca_enabled {
     # VPN Server or Client with external CA
-    if $extca_ca_cert_file == undef { fail('extca_ca_cert_file has to be specified in extca mode') }
-    if $extca_ca_crl_file == undef and $crl_verify and !$remote { fail('extca_ca_crl_file has to be specified in extca mode if crl_verify is enabled') }
-    if $extca_server_cert_file == undef { fail('extca_server_cert_file has to be specified in extca mode') }
-    if $extca_server_key_file == undef { fail('extca_server_key_file has to be specified in extca mode') }
-    if $extca_dh_file == undef and !$remote and $tls_server { fail('cant enable tls_server: missing extca_dh_file') }
-    if $extca_tls_auth_key_file == undef and !$remote and $tls_auth { fail('cant enable tls_auth: missing extca_tls_auth_key_file') }
+    if $extca_ca_cert_file == undef {
```

Comment:
How do you like this?

```puppet
unless $extca_ca_cert_file {}
```

 This is maybe easier to read?

Reply:
I prefer that syntax myself, though I know others don't. I'm wondering if those sort of changes are best made after we at least get the tests passing w/ the updated code, and maybe in a separate PR?

Reply:
I was actually having the exact same thought, though, when I was rebasing it.

Reply:
yeah, we should not clutter up this PR.